app/models/user_email.rb

```diff
@@ -5,6 +5,7 @@ class UserEmail < ActiveRecord::Base
 
   attr_accessor :skip_validate_email
   attr_accessor :skip_validate_unique_email
+  attr_accessor :skip_normalize_email
```

Comment:
It won’t change much, but instead of using `attr_accessor` on AR models, we could use `attribute`, like this for example:
```ruby
attribute :skip_normalize_email, :boolean, default: false
```
AR will know this is a virtual attribute and won’t try to save it to the DB. Up to you, though :slightly_smiling_face:

Reply:
In this case it's intentional. I don't want this included if calling `user_email.attributes`, etc. 🙂 It's a little post-it note pasted on the document, but not part of the document itself.